courses/machine_learning/deepdive2/launching_into_ml/solutions/explore_data.ipynb

```diff
@@ -0,0 +1,1980 @@
+{
```

Comment:
<p>I'm not too familiar with seaborn library. What it be possible to get start or end markers? Or some way to mark those with a toll vs wthout? It's a little hard for me tell how this fits in the narrative.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Maybe, this is what we've always had. The main point of this is to show that normal trips are fairly random and can have small distances (you see that little dot there and a couple short lines) while the tollrides usually have long distances and in a general direction (because he wants to pay a toll to drive a quarter mile?!). I left it as is for now and we can touch up later. :)</p>

Reply:
Ok, I think I'm following a little better. May I propose one quick changes though? Trips (in total) still includes rides with tolls. Can we make a new df for trips without tolls and plot that to make this a little clearer?

Reply:
Done